group_vars/all.yml

```diff
@@ -167,6 +167,11 @@ shift_stack: false
 #installs browbeat, if this variable is not defined or set true
 #skips browbeat installation, only if the variable set false
 browbeat: true
+
+# define mount_nvme to mount /var/lib/nova on nvme device
+# to use nvme storage for nova instances as ephemeral disk
+#mount_nvme: true
```

Comment:
I recommend mount_nvme to be a dictionary with one of the keys being the path of the nvme device (/dev/nvmen01..). Similarly for the passthrough stuff, we should change the name from `nvme` to something like `passthrough_nvme` to distinguish

Reply:
we can think about using as dict later when we get other requirements

Reply:
@venkataanil can you add these variables in CI files too.